src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Navigation () {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id = 'hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "http:// bbb.testpro.io/registration.php";
+
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222059905-96ee2061-4e2f-4489-b751-5930a7a81a86.png)

![image](https://user-images.githubusercontent.com/104761482/222059987-057f1b5f-b129-4b67-8fd1-3074677a518b.png)